etc/run-oidc-test.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
I made this an executable rather than using `go test` since this is what gssapi is doing... but also I know that when I go to test GCP and Azure that I'm going to want to build ahead of time, and I'm not sure if `go test` can build an executable (`cargo test` can).